.gitlab-ci.yml

```diff
@@ -3,8 +3,7 @@ image: stackstate/stackstate-agent-runner-circle:latest
 stages:
 - prepare
```

Comment:
Theres a couple of environment variables that are not here:

https://gitlab.com/stackvista/stackstate-agent/settings/ci_cd

but should go in this file like here:

https://docs.gitlab.com/ee/ci/yaml/README.html#variables

PROCESS_AGENT_VERSION: Should be in the release.lock
To gitlab file:
STS_AWS_RELEASE_BUCKET, STS_AWS_TEST_BUCKET, STS_DOCKER_RELEASE_REPO, STS_DOCKER_TEST_REPO

Reply:
@craffit release.lock , what you mean ?

Reply:
@TizianoPerrucci I meant this file: https://github.com/StackVista/stackstate-agent/blob/master/release.json. That should determine the versions we build against

Reply:
interesting, I even didn't notice that file :sweat_smile: 
I kinda like your suggestion, but don't think this PR is the place where we should start using the release.json
lets get another ticket :) ?

Reply:
I think, it get's injected already

Reply:
Okidok, we do this some other time